dags/catalyst.py

```diff
@@ -0,0 +1,117 @@
+"""
+DAG to schedule generation of performance reports for recently completed nimbus experiments.
+
+See the [catalyst repository](https://github.com/mozilla/catalyst).
+
+*Triage notes*
+
+This app will perform some bigquery queries, and generate statistical reports based on that data which are
+then published to https://protosaur.dev/perf-reports/index.html.
+
+Generally, there should be minimal triage necessary for failures unless it's related to infrastructure issues.
+Any failures related to the app execution itself will be taken care of directly by the performance team.
+
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.sensors.external_task import ExternalTaskSensor
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.tags import Tag
+
+default_args = {
+    "owner": "dpalmeiro@mozilla.com",
+    "email": [
+        "dpalmeiro@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "start_date": datetime(2025, 3, 18),
```

Comment:
I believe Airflow will try to catch up by running all the intervening dates, so to be safe let's go with today.
```suggestion
    "start_date": datetime(2025, 4, 30),
```